src/azure-cli/azure/cli/command_modules/apim/_help.py

```diff
@@ -612,3 +617,30 @@
         az apim api schema wait --created --api-id MyApi --name MyApim --schema-id schemaId -g MyResourceGroup
     crafted: true
 """
+
+helps['apim deletedservice show'] = """
+type: command
+short-summary: Get soft-deleted Api Management service instances available for undelete by name.
+examples:
+  - name: Get a soft-deleted services with its name.
+    text: |
+        az apim deletedservice get --name MyApim -g MyResourceGroup
+"""
+
+helps['apim deletedservice list'] = """
+type: command
+short-summary: Lists all soft-deleted Api Management services instances available for undelete for the given subscription.
+examples:
+  - name: List all soft-deleted services in a subscription.
+    text: |
+        az apim deletedservice list -g MyResourceGroup
```

Comment:
```suggestion
        az apim deletedservice list
```

Reply:
I recommend not adding `-g MyResourceGroup` here.
Otherwise, it is inconsistent with the example name.